### PR TITLE
feat(bitgo): sdk script to get token balance from wallet

### PR DESCRIPTION
**Pull request by margueriteblair**
A script to get token balances from a wallet. We currently only have a script that can get native asset balances.
Ticket: BG-62315

### CODE REVIEW

**Review by alia-bitgo — APPROVED** [review]
lgtm :)

File: modules/sdk-core/src/bitgo/wallet/iWallet.ts [hunk 1]
```diff
@@ -497,6 +497,7 @@ export interface WalletData {
   };
   multisigType: 'onchain' | 'tss';
   type?: WalletType;
+  tokens: Record<string, any>[];
```
Comment (mmcshinsky-bitgo) [inline modules/sdk-core/src/bitgo/wallet/iWallet.ts:500]: Will a tokens array always be returned all all coin wallets?

Comment (margueriteblair) [reply modules/sdk-core/src/bitgo/wallet/iWallet.ts:500]: craaap good catch, forgot to make optional

File: modules/sdk-core/src/bitgo/wallet/wallet.ts [hunk 2]
```diff
@@ -279,6 +279,13 @@ export class Wallet implements IWallet {
     return this._wallet.receiveAddress.address;
   }
 
+  /**
+   * Get a list of tokens and their balances from a wallet
+   */
+  public tokens(): Record<string, any>[] | undefined {
```
Comment (mmcshinsky-bitgo) [inline modules/sdk-core/src/bitgo/wallet/wallet.ts:285]: To make sure, is this a convenience function over `toJSON` or is there another intent as well?

Comment (margueriteblair) [reply modules/sdk-core/src/bitgo/wallet/wallet.ts:285]: it's a getter for the tokens because they're not accessible otherwise

Comment (margueriteblair) [reply modules/sdk-core/src/bitgo/wallet/wallet.ts:285]: Same way the functions above operate pretty much :)

Comment (mmcshinsky-bitgo) [reply modules/sdk-core/src/bitgo/wallet/wallet.ts:285]: But I can also do that via `toJson`, yes?

Comment (margueriteblair) [reply modules/sdk-core/src/bitgo/wallet/wallet.ts:285]: oh shoot, yeah you can i see, hadnt seen that in the wallet class before, so i guess it is entirely just a convenience function then

Comment (margueriteblair) [reply modules/sdk-core/src/bitgo/wallet/wallet.ts:285]: happy to change that if needed!

Comment (mmcshinsky-bitgo) [reply modules/sdk-core/src/bitgo/wallet/wallet.ts:285]: Yeah lets use what we have for now. Thanks!

Comment (michaelmcshinsky) [reply modules/sdk-core/src/bitgo/wallet/wallet.ts]: You can still leave the tokens type on there. Will help for clarity and documentation

**Pull request merged**